sotodlib/tod_ops/pca.py

```diff
@@ -300,3 +303,53 @@ def pca_cuts_and_cal(tod, pca_aman, xfac=2, yfac=1.5, calc_good_medianw=False):
     pca_relcal.wrap('median', medianw)
 
     return pca_relcal
+
+
+def get_common_mode(
+    tod,
+    signal='signal',
+    method='median',
+    wrap_name=None,
+    weights=None,
+):
+    """Returns common mode timestream between detectors.
+    This uses method 'median' or 'average' across detectors as opposed to a principle
+    component analysis to get the common mode.
+
+    Arguments
+    ---------
+        tod: axis manager
+        signal: str, optional
+            The name of the signal to estimate common mode or ndarray with shape of
+            (n_dets x n_samps). Defaults to 'signal'.
+        method: str
+            method of common mode estimation. 'median' or 'average'.
+        wrap_name: str or None.
+            If not None, wrap the common mode into tod with this name.
```

Comment:
The convention in this module is to use "wrap" for the name of this argument.  Please make consistent.